src/styles/blocks/header.scss

```
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: $transition-card-duration;
```

You should specify  the transition property
```suggestion
    transition: color $transition-card-duration;
```